src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
```
потрібно використовувати абстрактний тип, так можна буде використати поліморфізм якщо ці машини поставити в один масив наприклад і циклом по ньому пробігтись